### PR TITLE
Add on to AddressBookParserTest

### PR DESCRIPTION
**Pull request by jiale-c**
Add test methods for filter and summarise for AddressBookParserTest as they were missing prior to this PR. This is a set of  more tests to all methods and classes to ensure code coverage of the files increase.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SummariseCommand.java [hunk 1]
```diff
@@ -165,7 +165,7 @@ private String summariseBlock(List<Person> result, String blockLetter) {
      * @param list the unfiltered entire list in the database
```
Comment (DannyDakota) [inline src/main/java/seedu/address/logic/commands/SummariseCommand.java:165]: Can correct this the sentence a bit weird

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java [hunk 2]
```diff
@@ -14,21 +14,27 @@
...
 import seedu.address.logic.parser.exceptions.ParseException;
 import seedu.address.model.person.NameContainsKeywordsPredicate;
 import seedu.address.model.person.Person;
 import seedu.address.testutil.EditPersonDescriptorBuilder;
 import seedu.address.testutil.PersonBuilder;
 import seedu.address.testutil.PersonUtil;
 
+/**
+ * Contains helper methods for testing address book parsers.
+ */
```
Comment (DannyDakota) [inline src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:37]: Thank you for describing the series of test cases!

**Review by DannyDakota — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/logic/commands/SummariseCommand.java (hunk 1 above)
Comment (jiale-c) [reply src/main/java/seedu/address/logic/commands/SummariseCommand.java]: Hi! Thank you for pointing that out. I guess it was hard to understand. I have simplified the description of this param found in the rest of the methods too. Thank you. :stuck_out_tongue_closed_eyes:

Comment (DannyDakota) [reply src/main/java/seedu/address/logic/commands/SummariseCommand.java]: Fantastic! Thank you jiale 😄

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java [hunk 3]
```diff
@@ -88,6 +94,27 @@ public void parseCommand_list() throws Exception {
...
+    public void parseCommand_archive() throws Exception {
+        assertTrue(parser.parseCommand(ArchiveCommand.COMMAND_WORD) instanceof ArchiveCommand);
+        assertTrue(parser.parseCommand(ArchiveCommand.COMMAND_WORD + " 3") instanceof ArchiveCommand);
+    }
+
+    @Test
+    public void parseCommand_filter() throws Exception {
+        FilterCommand.FilterDescriptor filterDescriptorTest = new FilterCommand.FilterDescriptor();
+        List<String> keywords = Arrays.asList("foo", "bar", "baz");
+        FilterCommand command2 = (FilterCommand) parser.parseCommand(
```
[the remark this answers is not in the document]
Comment (jiale-c) [reply src/test/java/seedu/address/logic/parser/AddressBookParserTest.java]: I will stick to just command. it was a typo that I missed. Thanks for pointing it out!

**Review by DannyDakota — APPROVED** [review]
LGTM!! 💯

**Pull request merged**